.travis.yml

```diff
@@ -12,26 +12,10 @@ env:
     - DESTINATION="OS=8.2,name=iPad Air" SDK=iphonesimulator9.3
```

Comment:
Change to add a type:

- DESTINATION="OS=8.2,name=iPad Air" SDK=iphonesimulator9.3 TYPE="Unit"
- DESTINATION="OS=8.2,name=iPad Air" SDK=iphonesimulator9.3 TYPE="FunctionalObjC"
- DESTINATION="OS=8.2,name=iPad Air" SDK=iphonesimulator9.3 TYPE="FunctionalSwift"
- DESTINATION="OS=8.2,name=iPad Air" SDK=iphonesimulator9.3 TYPE="Demo"
- TYPE="Ruby"

You can then use the TYPE to switch on the tests that you want to run. Notice how with the `elif`, for 9.2 for eg. only the Unit Tests will be run when we want all possible tests to be run. Using the above matrix change, we can get independent machines running each test since each will be a different run in the matrix.